.github/workflows/windows.yml

```diff
@@ -20,4 +20,4 @@ jobs:
         $Env:GOPATH = "$Env:GITHUB_WORKSPACE"
         cd "$Env:GITHUB_WORKSPACE"
         cd "src/github.com/aws/amazon-ecs-agent"
-        go test -v -race -tags unit -timeout 40s ./agent/...
+        go test -race -tags unit -timeout 40s ./agent/...
```

Comment:
Why did we remove the `-v` ?

Reply:
By removing this flag, we are able to remove the excess noise from the UTs during PR. We can very easily pin point the UT which failed and the corresponding error. For example - https://github.com/aws/amazon-ecs-agent/runs/2786834601?check_suite_focus=true

Also, this ensures that the behavior is same for Linux and Windows UT during PR build.

Reply:
Nice 👍